docs/conceptual/array.sum[^t]-function-[fsharp].md

```diff
@@ -36,15 +36,16 @@ The input array.
 
 
 ## Return Value
-
-The resulting sum.
+The sum of all elements of the array.
 
 
 ## Exceptions
-May throw an [OverflowException](https://msdn.microsoft.com/library/system.overflowexception.aspx), as the function uses the [checked (+) operator](https://msdn.microsoft.com/visualfsharpdocs/conceptual/checked.[-p-][%5Et1%2c%5Et2%2c%5Et3]-function-[fsharp]) for the additions.
+May throw an [OverflowException](https://msdn.microsoft.com/library/system.overflowexception.aspx) due to arithmetic overflows.
```

Comment:
The Remarks section is right below this, where I moved the link to the checked operator to. The link is in bold and it's highly likely that it'll be in view. (This is consistent with the way it was documented on the `Seq` module, except that it wasn't a link there. The line I'm moving on this one is from my last commit, from when I hadn't looked at all the variants yet.)